Assets/_Game/Scripts/Wave/TimeManager.cs

```diff
@@ -105,7 +108,7 @@ public class TimeManager : Singleton<TimeManager> {
 
   private void SpawnEnemies() {
     int numEnemies = currentWave * waveDataLoader.numEnemiesPerWave * currentSubWave;
-    for (int i = 0 ; i < numEnemies ; i++) {
+    for (int i = 0; i < numEnemies; i++) {

```

Comment:
We should avoid using Coroutine inside a loop. We can use only one Coroutine to spawn enemies per subway/way.